packages/framework/src/Resources/scripts/admin/sideMenu.js

```diff
@@ -0,0 +1,24 @@
+(function ($) {
```

Comment:
This commit ( 361fc88a84045b47a74978f243bb2f2a95694742 ) should be before the one with documentation

Reply:
It is (and was). GitHub displays the commits in different order, try fetching it and running `git ll` locally